aws-auth-cognito/src/test/java/com/amplifyframework/auth/cognito/RealAWSCognitoAuthPluginTest.kt

```diff
@@ -1052,4 +1054,70 @@ class RealAWSCognitoAuthPluginTest {
         )
         coVerify(exactly = 0) { onError.accept(any()) }
     }
+
+    @Test
+    fun `confirm user attribute fails when not in SignedIn state`() {
```

Comment:
Can we add a test for when cognito throws an error?

Reply:
yes done